c7n/resources/awslambda.py

```diff
@@ -255,6 +293,75 @@ class KmsFilter(KmsRelatedFilter):
     RelatedIdsExpression = 'KMSKeyArn'
 
 
+@AWSLambda.action_registry.register('set-enhanced-monitoring')
```

Comment:
yes, you're right, shall I change this action to "inject-layer", so that user can use it, in both the ways? and make lambda-layer-arn as mandatory value?

Reply:
sounds good to me

Reply:
ok, Changes done!